src/core/config/tabular_data/crud_operations/update_in_place/option.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "config/common_option.h"
+#include "config/equal_nulls/type.h"
+
+namespace config {
+extern CommonOption<bool> const kUpdateInPlaceOpt;
```

Comment:
Is it really an option that can be reused?

Reply:
Yes, for example: DynamicFD algo needs to represent update operation as combination of delete and insert (insert to the end of table), and DynamicFDVerifier doesn't need. It's even better to to in-place update here, because it will be simpler and clearer to user, if we will process update as a single operation